paw-jieba.el

```diff
@@ -0,0 +1,106 @@
+(require 'jieba)
```

Comment:
Would you check if jieba is in elpa or melpa, if so, please put it into paw.el Package-Requires section. If not, use `(require jieba nil t)` to avoid missing package error.

Reply:
Done. It's not on Melpa or Elpa